mne/io/proj.py

```diff
@@ -779,14 +779,17 @@ def deactivate_proj(projs, copy=True, verbose=None):
 
 
 @verbose
-def make_eeg_average_ref_proj(info, activate=True, verbose=None):
+def make_eeg_average_ref_proj(info, activate=True, ch_type=('eeg',),
+                              verbose=None):
     """Create an EEG average reference SSP projection vector.
 
     Parameters
     ----------
     %(info_not_none)s
     activate : bool
         If True projections are activated.
+    ch_type : tuple
+        List of channel types to use for reference projection.
```

Comment:
@moritz-gerster can you list the valid channel types. I would say this makes sense for EEG, iEEG and ECoG?

Reply:
Yes, @agramfort but also for DBS! We use the DBS leads to measure local field potentials.